tasks/eva_2038/storeSSIDsInVariantWarehouse.py

```diff
@@ -0,0 +1,162 @@
+
+
+@retry(tries=3, delay=2, backoff=1.5, jitter=(1, 3))
+def download_file_from_ftp(ftp, file, local_file_path):
+    logger.info(f"Trying to download file: {file}")
+    with open(local_file_path, 'wb') as local_file:
+        ftp.retrbinary(f"RETR {file}", local_file.write)
+
+
+def run_import_accession_job_for_project(working_dir, params, project_id):
+    logger.info(f"Starting processing project: {project_id}")
+
+    project_path = os.path.join(params['project_dir'], project_id)
+    if os.path.exists(project_path):
+        accession_report_files = [
+            os.path.join(project_path, '60_eva_public', f)
+            for f in os.listdir(os.path.join(project_path, '60_eva_public'))
+            if ".accessioned.vcf" in f]
+    else:
+        logger.warning(f"Could not find the Project {project_id} in project_path {project_path}. "
+                       f"Trying to retrieve accession report files from FTP")
+        try:
+            accession_report_files = get_accession_report_files_from_ftp(working_dir, project_id)
+        except Exception:
+            raise Exception(f"Error fetching files from ftp for study {project_id}.")
+
+    if not accession_report_files:
+        raise Exception(f"No accession report files found for project {project_id} in CODON/FTP")
+    else:
+        formatted_name_file_dict = {
+            os.path.basename(file).replace('.accessioned.vcf', '.vcf'): file
+            for file in accession_report_files
+        }
+
+        file_asm_tax_list = get_tax_asm_details(params, project_id, formatted_name_file_dict.keys())
+        logger.info(f"acc report files and their corresponding asm accession and taxonomy_id : {file_asm_tax_list}")
+
+        if len(file_asm_tax_list) != len(accession_report_files):
+            missing_files = set(formatted_name_file_dict.keys()) - set([file_asm_tax_list[i][0] for i in range(len(file_asm_tax_list))])
+            raise Exception(
+                f"For project {project_id}, File mismatch between DB and Codon/FTP."
+                f"\nFiles missing: {[os.path.basename(formatted_name_file_dict[f]) for f in missing_files]}")
+        else:
+            os.makedirs(os.path.join(working_dir, project_id), exist_ok=True)
+            with get_metadata_connection_handle("production_processing", params['private_settings_xml_file']) as pg_conn:
+                acc_file_db_name_csv = os.path.join(working_dir, project_id, 'acc_file_db_name.csv')
+                with open(acc_file_db_name_csv, 'w', newline='') as file:
+                    writer = csv.writer(file)
+                    writer.writerow(['acc_report_file', 'db_name'])
+                    for acc_file, asm, taxonomy in file_asm_tax_list:
+                        db_name = resolve_variant_warehouse_db_name(pg_conn, asm, taxonomy)
+                        writer.writerow([formatted_name_file_dict[acc_file], db_name])
+
+                run_nextflow(working_dir, params, project_id, acc_file_db_name_csv)
```

Comment:
```suggestion
            run_nextflow(working_dir, params, project_id, acc_file_db_name_csv)
```
since the db connection isn't needed anymore

Reply:
done